src/main/resources/l10n/JabRef_en.properties

```diff
@@ -1325,11 +1321,14 @@ All\ key\ bindings\ will\ be\ reset\ to\ their\ defaults.=All key bindings will
 
 Automatically\ set\ file\ links=Automatically set file links
 Resetting\ all\ key\ bindings=Resetting all key bindings
-Hostname=Hostname
-Invalid\ setting=Invalid setting
+
 Network=Network
-Please\ specify\ both\ hostname\ and\ port=Please specify both hostname and port
-Please\ specify\ both\ username\ and\ password=Please specify both username and password
+Search\ IEEEXplore=Search IEEEXplore
```

Comment:
Normally it should be sufficient to have the Search %0. It would be interesting to see if this works in fxml somewhow.

Reply:
I tried around a little bit, but I could not find a real solution. The problem is in mixing Ressource-Resolution and dynamic binding in one property. The other option would be to put setText(Localization.lang("Search %0") + "IEEExplore") into the Java-View.